source/mir/algorithm/iteration.d

```diff
@@ -20,8 +20,10 @@ $(T2 find, Finds backward index.)
 $(T2 findIndex, Finds index.)
 $(T2 fold, Accumulates all elements (different parameter order than `reduce`).)
 $(T2 isSymmetric, Checks if the matrix is symmetric.)
+$(T2 maxElement, Returns the maximum.)
 $(T2 maxIndex, Finds index of the maximum.)
 $(T2 maxPos, Finds backward index of the maximum.)
+$(T2 maxIndex, Returns the minimum.)
```

Comment:
L26: this should read minElement, not maxIndex

Reply:
@prbuen Apologies on the delay. Fixed now.